### ISSUE TITLE
Ignore external imports on build if they are not found

### ISSUE THREAD

**Issue by razvanmazilu0** [ionic-team/stencil#4736]
**Prerequisites**

 - [x] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/.github/CONTRIBUTING.md).
 - [x]  I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
 - [x]  I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already include this feature request, without success.

**Stencil version:**

```
 @stencil/core@<2.11.0">
```

**Current behavior:**

Hello Iconic Team,

We are currently trying to build multiple web components (made with StencilJS) in a bigger project, which it's incorporating all of them.

Problem: These components have external imports (see examples below), which are crashing when we try to build the component individually. Unfortunately we cannot get rid of these imports. Is there any possibility using only **stencil.config.ts**,  to mark all the external dependencies to be ignored at build time and treat them as it is? 

imports example: `import { C_CONST } from '../../../../../my-services';`

Error on build:
```
 [ ERROR ]  Rollup: Unresolved Import 
           Could not resolve '../../../../../my-services' from ./src/components/my-component/my-component.tsx 
```
Basically we want to use the **external** property configuration of the Rollup plugin built in the Stencil Core. https://rollupjs.org/configuration-options/#external

Also if you can provide an concluding example  with  **stencil.config.ts** will be appreciated.

**Comment by razvanmazilu0** [comment]
Hello,

Added a link with a demo component in order to be easier to reproduce: ttps://github.com/MarianTerinte96/stencilTestApp/tree/main/stenciltestapp

Here is the external imports which are problematic: https://github.com/MarianTerinte96/stencilTestApp/blob/main/stenciltestapp/src/components/my-component/my-component.tsx#L2-L4

**Comment by rwaskiewicz** [comment]
Thanks @razvanmazilu0! I believe that is this a duplicate of https://github.com/ionic-team/stencil/issues/3226. As a result, I'm going to close this out. Thanks!
